src/backend/queue_agent/main.py

```diff
@@ -96,7 +96,11 @@ def main():
                 try:
                     snsPayload = json.loads(message.body)
                     payload = json.loads(snsPayload['Message'])
-                    taskHeader = payload.pop('alwayson_scripts', None)
+                    taskHeader = copy.deepcopy(payload).get('alwayson_scripts', None)
+                    alwayson_scripts = {k: v for k, v in taskHeader.items() if
+                                        k not in ["task", "sd_model_checkpoint", "id_task", "uid", "save_dir"]}
```

Comment:
I would suggest for ignoring more keys like `image_link` ,  `sd_vae` due to DBT customized payload.

Reply:
We fix it with more compatible changes in PR #20 , which means this PR will be closed.